frontend/src/pages/Molecule.jsx

```diff
@@ -207,7 +212,10 @@ export default function MoleculeInfo() {
                         ))}
                      </Select>
                   </FormControl>
-                  <Box display="flex" justifyContent="center" alignItems="center">
+                     <Box display="flex" justifyContent="center">
```

Comment:
This is identical to the button above which is inside the molecule info section. Is this deliberately done?

Reply:
No, oops! I didn't mean to include this. I was playing around with the idea of a "Download SDF" button, but it was more work than I wanted to put on this PR. Deleting this button.